### ISSUE TITLE
Should godot use wildcard in csproj?

### ISSUE THREAD

**Issue by Thaina** [godotengine/godot-proposals#1565]
**Describe the project you are working on:**
With any project I always use C# script

**Describe the problem or limitation you are having in your project:**
Adding and pasting cs file are not include in godot build

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
I think csproj are capable of using wildcard include so didn't godot should generate csproj with wildcard by default?

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
```xml
    <ItemGroup>
        <CSFile Include="*.cs"/>
    </ItemGroup>
```

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
Not sure about this question

**Is there a reason why this should be core and not an add-on in the asset library?:**
It should be default to every project

**Comment by DarkKilauea** [comment]
As of the latest stable release 3.2.3, Godot already includes .cs files by wildcard via its SDK project file.  I believe your proposal has already been implemented.

**Comment by neikeq** [comment]
Yup, this is already implemented in 3.2.3. You can get a similar effect in older versions with `<Compile Include="**\*.cs" />`.
